src/matlab/reactormethods.cpp

```diff
@@ -48,6 +48,13 @@ void reactormethods(int nlhs, mxArray* plhs[],
         case 7:
             iok = reactor_setKineticsMgr(i, int(v));
             break;
+        case 8:
+            // This casting of double to bool is recommended by the MathWorks
```

Comment:
Sure, using a `double` as a `bool` isn't normally done in C++, which is the reason for writing the cast explicitly: to show that you really mean it instead of relying on an implicit conversion. I'm actually not entirely sure that the advice on that page applies -- the value passed in to `ctmethods` really is a `double`, not a `bool` that's been cast to a `double`.